showkase-browser-testing/src/androidTest/java/com/airbnb/android/showkase_browser_testing/ShowkaseBrowserTest.kt

```diff
@@ -37,31 +36,54 @@ class ShowcaseBrowserTest {
             }
         )
 
+    // This will alter now since KSP supports stacked preview annotations and KAPT does not.
+    private val componentSize = if (System.getProperty("useKsp") == "true") {
```

Comment:
Can you add a more descriptive comment about why we are doing this. Along with link to the issue

Reply:
Good idea! Added something in [fb87f3c](https://github.com/airbnb/Showkase/pull/259/commits/fb87f3cf14301ee3f72b288ce9955b59b0d32b1b) :) Does that look okay? :)